tsconfig.base.json

```
@@ -0,0 +1,9 @@
+{
+	"extends": "@skyra/eslint-config",
+	"compilerOptions": {
+		"lib": ["ESNext", "DOM"],
```

```suggestion
		"lib": ["ESNext"],
```

This lib doesn't need DOM